src/pybind/mgr/dashboard/frontend/src/testing/unit-test-helper.ts

```diff
@@ -101,3 +104,60 @@ export class PermissionHelper {
     });
   }
 }
+
+export class FormHelper {
+  form: CdFormGroup;
+
+  constructor(form: CdFormGroup) {
+    this.form = form;
+  }
+
+  private getControl(control: AbstractControl | string): AbstractControl {
+    if (typeof control === 'string') {
+      return this.form.get(control);
+    }
+    return control;
+  }
+
+  hasError(control: AbstractControl | string, error: string) {
+    expect(this.getControl(control).hasError(error)).toBeTruthy();
+  }
+
+  isValid(control: AbstractControl | string) {
+    expect(this.getControl(control).errors).toBe(null);
+  }
+
+  setValue(control: AbstractControl | string, value: any, markAsDirty?: boolean): AbstractControl {
+    control = this.getControl(control);
+    control.setValue(value);
+    if (markAsDirty) {
+      control.markAsDirty();
+    }
+    return control;
+  }
+
+  isValidChange(control: AbstractControl | string, value: any) {
+    this.isValid(this.setValue(control, value));
+  }
+
+  isErrorChange(control: AbstractControl | string, value: any, error: string) {
+    this.hasError(this.setValue(control, value), error);
+  }
+
+  setMultipleValues(values: object, markAsDirty?: boolean) {
+    Object.keys(values).forEach((key) => {
+      this.setValue(key, values[key], markAsDirty);
+    });
+  }
+
+  isElementVisible(fixture: ComponentFixture<any>, css: string, visibility: boolean) {
```

Comment:
I'm torn about this one as it's not used anywhere. The same applies to `areIdElementsVisible`. Wouldn't it make more sense to implement such features when are actually being used?